mitodl/mitxpro
# fix: make the course topics selection optional in CMS

#### Pre-Flight checklist


#### What are the relevant tickets?
None, We recently added topics filtering feature through https://github.com/mitodl/mitxpro/pull/2609. With that, we added a topics field in CMS pages but currently, it's a required field and it's possible that a course might not be associated with any topic based on current data.

#### What's this PR do?
Markes the topics field `Optional`.


#### How should this be manually tested?
- Setup master, create an external or internal course, and check that the topics field on CMS page is required.
- Move to this branch, Make sure you can save the pages without adding a topic to the page.


#### Screenshots (if appropriate)
**After**

<img width="1381" alt="image" src="https://user-images.githubusercontent.com/34372316/232471448-86fe335f-c0ac-47e6-81e5-6d72b47cce61.png">

**Before**

<img width="1019" alt="image" src="https://user-images.githubusercontent.com/34372316/232471547-454c24f1-644a-4d4b-b4dd-1502e2cbef5c.png">
